second-edition/src/SUMMARY.md

```diff
@@ -1,131 +1,131 @@
-    - [Controlling Visibility with `pub`](ch07-02-controlling-visibility-with-pub.md)
-    - [Importing Names with `use`](ch07-03-importing-names-with-use.md)
+- [Módulos](ch07-00-modules.md)
+    - [`mod` e o Sistema de Arquivos](ch07-01-mod-and-the-filesystem.md)
+    - [Controlando Visibilidade com `pub`](ch07-02-controlling-visibility-with-pub.md)
+    - [Importando Nomes com `use`](ch07-03-importing-names-with-use.md)
 
-- [Common Collections](ch08-00-common-collections.md)
-    - [Vectors](ch08-01-vectors.md)
+- [Coleções Comuns](ch08-00-common-collections.md)
+    - [Vetores](ch08-01-vectors.md)
     - [Strings](ch08-02-strings.md)
     - [Hash Maps](ch08-03-hash-maps.md)
 
-- [Error Handling](ch09-00-error-handling.md)
-    - [Unrecoverable Errors with `panic!`](ch09-01-unrecoverable-errors-with-panic.md)
-    - [Recoverable Errors with `Result`](ch09-02-recoverable-errors-with-result.md)
-    - [To `panic!` or Not To `panic!`](ch09-03-to-panic-or-not-to-panic.md)
+- [Tratando Erros](ch09-00-error-handling.md)
+    - [Erros Irrecuperáveis com `panic!`](ch09-01-unrecoverable-errors-with-panic.md)
+    - [Erros Recuperáveis com `Result`](ch09-02-recoverable-errors-with-result.md)
+    - [Entrar em `panic!` ou Não Entrar em `panic!`](ch09-03-to-panic-or-not-to-panic.md)
 
-- [Generic Types, Traits, and Lifetimes](ch10-00-generics.md)
-    - [Generic Data Types](ch10-01-syntax.md)
-    - [Traits: Defining Shared Behavior](ch10-02-traits.md)
-    - [Validating References with Lifetimes](ch10-03-lifetime-syntax.md)
+- [Tipos Genéricos, Traits, e Lifetimes](ch10-00-generics.md)
+    - [Dados de Tipos Genéricos](ch10-01-syntax.md)
+    - [Traits: Definindo Comportamento Compartilhado](ch10-02-traits.md)
+    - [Validando Referências com  Lifetimes](ch10-03-lifetime-syntax.md)
 
-- [Testing](ch11-00-testing.md)
-    - [Writing tests](ch11-01-writing-tests.md)
-    - [Running tests](ch11-02-running-tests.md)
-    - [Test Organization](ch11-03-test-organization.md)
+- [Testes](ch11-00-testing.md)
+    - [Escrevendo Testes](ch11-01-writing-tests.md)
+    - [Executando Testes](ch11-02-running-tests.md)
+    - [Organização de Teste](ch11-03-test-organization.md)
 
-- [An I/O Project: Building a Command Line Program](ch12-00-an-io-project.md)
-    - [Accepting Command Line Arguments](ch12-01-accepting-command-line-arguments.md)
-    - [Reading a File](ch12-02-reading-a-file.md)
-    - [Refactoring to Improve Modularity and Error Handling](ch12-03-improving-error-handling-and-modularity.md)
-    - [Developing the Library’s Functionality with Test Driven Development](ch12-04-testing-the-librarys-functionality.md)
-    - [Working with Environment Variables](ch12-05-working-with-environment-variables.md)
-    - [Writing Error Messages to `stderr` Instead of `stdout`](ch12-06-writing-to-stderr-instead-of-stdout.md)
+- [Projeto de E/S: Criando um Programa de Linha de Comando](ch12-00-an-io-project.md)
```

Comment:
Acho que "I/O" fica melhor, e ai no capítulo a gente coloca "In/Out (Entrada/Saída)" em algum momento.